plonky2/src/hash/poseidon.rs

```diff
@@ -365,6 +438,45 @@ pub trait Poseidon: PrimeField64 {
         result
     }
 
+    #[inline(always)]
+    #[unroll_for_loops]
+    fn mds_partial_layer_packed_init<
```

Comment:
Very nit: for naming consistency (and because it would be easier to find for users, I think)
```suggestion
    fn mds_partial_layer_init_packed_field<
```